### PR TITLE
vertically align channel contents and reverse order

### PR DESCRIPTION
**Pull request by billbitt**

### CODE REVIEW

File: client/src/utils/createGroupedList.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+export const createGroupedList = (list, size) => {
+  if (!size) {
+    console.log('no size provided to createGroupedList');
```
Comment (seanyesmunt) [inline client/src/utils/createGroupedList.js]: Not needed.

File: client/src/utils/createGroupedList.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    return list;
+  }
+  if (!list) {
+    console.log('no list provided to createGroupedList');
```
Comment (seanyesmunt) [inline client/src/utils/createGroupedList.js]: Same here

Comment (billbitt) [reply client/src/utils/createGroupedList.js]: my concern was I didn't want the function to return early silently.  I replaced the above console logs with errors.

**Comment by seanyesmunt** [comment]
Cool. Just squash your commits and it's good to go.

**Pull request merged**